src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,182 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.exception.InvalidUserAgeException;
+import core.basesyntax.exception.InvalidUserLoginException;
+import core.basesyntax.exception.InvalidUserPasswordException;
+import core.basesyntax.exception.UserAlreadyExistsException;
+import core.basesyntax.exception.UserValidationException;
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+
+    private RegistrationService registrationService = new RegistrationServiceImpl();
+    private User validUser = new User();
+
+    @BeforeEach
+    public void each() {
+        Storage.people.clear();
```

Comment:
```suggestion
        registrationService = new RegistrationServiceImpl();
        validUser = new User();
        Storage.people.clear();
```

Reply:
Nope, it won't work as you expect :(

Reply:
And we've discussed this line of code today on QnA with @yevheniisynytsia, he said it's ok